src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -604,24 +623,21 @@ func (c *controller) UploadLogs(ctx context.Context, cancellog context.CancelFun
 				}
 				podName = pods[0].Name
 			}
-			var err error
-			c.log.Info("Collecting Logs ....")
+
 			if status.HasError() {
-				//once error is detected in the flow, upload the full capacity of debugging logs
-				//and abort
-				c.log.Infof("Error detected... closing logs...")
+				c.log.Infof("Error detected. Closing logs and aborting...")
+				ticker.Stop()
```

Comment:
why we need ticker stop?

Reply:
For good measures to make sure it will not get selected again in the outer
select statement

Reply:
no need. till you are in case it will not be selected